File: 2745-construct-the-longest-new-string/2745-construct-the-longest-new-string.kt
class Solution {
    fun longestString(x: Int, y: Int, z: Int): Int {
        var ans = 0
        if (x == y) {
            ans = 2 * minOf(x, y)
        } else {
            ans = 2 * minOf(x, y) + 1
        }

        ans += z

        return ans * 2
    }
}

File: 2745-construct-the-longest-new-string/2745-construct-the-longest-new-string.py
class Solution(object):
    def longestString(self, x, y, z):
        """
        :type x: int
        :type y: int
        :type z: int
        :rtype: int
        """
        if x == y:
            ans = 2 * min(x,y)
        else: ans = 2 * min(x,y) + 1 
            
        ans += z

        return ans * 2         